test/addons-abi/test_buffer/test_buffer.cc

```diff
@@ -0,0 +1,141 @@
+#include <string.h>
+#include <string>
+#include <node_api_helpers.h>
+
+#define JS_ASSERT(env, assertion, message) \
```

Comment:
I'll file an issue to pass over the tests and use macros. I've distilled [a bunch of macros](https://github.com/gabrielschulhof/iotivity-node/blob/f09b6ae0c51d8b653bd32f51b72ac5d3b4c3ec73/src/common.h) that come in handy with this API, though those are more generic than we might need for the tests, because in the tests we're not likely to have helper functions.